.changeset/wicked-cups-tie.md

```diff
@@ -0,0 +1,8 @@
+---
```

Comment:
Added two changesets because this PR because this PR does two different things. Let's see how #596 consumes this.